api/handlers/ownerinfo.go

```diff
@@ -77,8 +77,8 @@ func createOwnerData(w http.ResponseWriter, r *http.Request, mu *sync.Mutex) {
 
 	slog.Debug("ownerData created")
 
-	w.WriteHeader(http.StatusCreated)
 	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(http.StatusCreated)
```

Comment:
applied https://github.com/fido-device-onboard/go-fdo-server/pull/32 to get the tests to pass.